### PR TITLE
create badge documentation

### PR DESCRIPTION
**Pull request by Mady-Camara**

### CODE REVIEW

File: Blazor_Foundation_6_Test_Shared/Documentation/Media/Badge.razor [hunk 1]
```diff
@@ -1,7 +1,64 @@
...
+        <h2>Badge</h2>
+        <p>The badge is a basic component that displays a number. It's useful for calling out a number of unread items.</p>
+        <hr />
+    </SectionLeft>
+</SectionWrapper>
+<SectionWrapper>
+    <SectionLeft>
+        <h2>Basics</h2>
+        <p>
+            Add the .badge class to an element to create a badge. In the below example, we're using <Code Content="PHNwYW4+"></Code>, but any tag will work fine.
```
Comment (maxgsamson) [inline Blazor_Foundation_6_Test_Shared/Documentation/Media/Badge.razor:15]: Add the .badge class should be in <Code> component

File: Blazor_Foundation_6_Test_Shared/Documentation/Media/Badge.razor [hunk 2]
```diff
@@ -1,7 +1,64 @@
...
+        </p>
+        <HighlightCS Language="html" Full="true" HeaderIcon="mdi-language-html5" HeaderBGcolor="#5d2b90" HeaderTitle="Razor"
+                     Content="PHNwYW4gY2xhc3M9ImJhZGdlIj4xPC9zcGFuPg==" />
+        <span class="badge">1</span>
+        <p>
+            A badge will typically be describing another element on the page. To bind the two elements together, give the badge a unique ID, and reference that ID in an aria-describedby attribute on the main element.
+        </p>
+        <HighlightCS Language="html" Full="true" HeaderIcon="mdi-language-html5" HeaderBGcolor="#5d2b90" HeaderTitle="Razor"
```
Comment (maxgsamson) [inline Blazor_Foundation_6_Test_Shared/Documentation/Media/Badge.razor:23]: No Visual Example?

File: Blazor_Foundation_6_Test_Shared/Documentation/Media/Badge.razor [hunk 3]
```diff
@@ -1,7 +1,64 @@
...
+                     Content="PGgxIGFyaWEtZGVzY3JpYmVkYnk9Im1lc3NhZ2VDb3VudCI+VW5yZWFkIE1lc3NhZ2VzPC9oMT4KPHNwYW4gY2xhc3M9ImJhZGdlIiBpZD0ibWVzc2FnZUNvdW50Ij4xPC9zcGFuPg==" />
+        <p>
+            Finally, the content itself might need more context for users that use screen readers. You can add extra text inside the badge using the .show-for-sr class.
+        </p>
+        <HighlightCS Language="html" Full="true" HeaderIcon="mdi-language-html5" HeaderBGcolor="#5d2b90" HeaderTitle="Razor"
```
Comment (maxgsamson) [inline Blazor_Foundation_6_Test_Shared/Documentation/Media/Badge.razor:28]: No Visual Example?

**Review by maxgsamson — CHANGES_REQUESTED** [review]
Check Comment Missing Visual Example

File: Blazor_Foundation_6_Test_Shared/Documentation/Media/Badge.razor (hunk 1 above)
Comment (Mady-Camara) [reply Blazor_Foundation_6_Test_Shared/Documentation/Media/Badge.razor:15]:
```suggestion
            Add the <Code>.badge</Code> class to an element to create a badge. In the below example, we're using <Code Content="PHNwYW4+"></Code>, but any tag will work fine.
```

File: Blazor_Foundation_6_Test_Shared/Documentation/Media/Badge.razor (hunk 2 above)
Comment (Mady-Camara) [reply Blazor_Foundation_6_Test_Shared/Documentation/Media/Badge.razor:23]:
```suggestion
<p>
            A badge will typically be describing another element on the page. To bind the two elements together, give the badge a unique ID, and reference that ID in an aria-describedby attribute on the main element.
        </p>
        <HighlightCS Language="html" Full="true" HeaderIcon="mdi-language-html5" HeaderBGcolor="#5d2b90" HeaderTitle="Razor"
                     Content="PGgxIGFyaWEtZGVzY3JpYmVkYnk9Im1lc3NhZ2VDb3VudCI+VW5yZWFkIE1lc3NhZ2VzPC9oMT4KPHNwYW4gY2xhc3M9ImJhZGdlIiBpZD0ibWVzc2FnZUNvdW50Ij4xPC9zcGFuPg==" />
        <h1 aria-describedby="messageCount">Unread Messages</h1>
        <span class="badge" id="messageCount">1</span>
```

File: Blazor_Foundation_6_Test_Shared/Documentation/Media/Badge.razor (hunk 3 above)
Comment (Mady-Camara) [reply Blazor_Foundation_6_Test_Shared/Documentation/Media/Badge.razor:28]:
```suggestion
 <p>
            Finally, the content itself might need more context for users that use screen readers. You can add extra text inside the badge using the .show-for-sr class.
        </p>
        <HighlightCS Language="html" Full="true" HeaderIcon="mdi-language-html5" HeaderBGcolor="#5d2b90" HeaderTitle="Razor"
                     Content="PHNwYW4gY2xhc3M9ImJhZGdlIiBpZD0ibWVzc2FnZUNvdW50Ij4xIAogICA8c3BhbiBjbGFzcz0ic2hvdy1mb3Itc3IiPgogICAgICB1bnJlYWQgbWVzc2FnZQogICA8L3NwYW4+Cjwvc3Bhbj4=" />
        <span class="badge" id="messageCount">
            1
            <span class="show-for-sr">
                unread message
            </span>
        </span>
```

File: Blazor_Foundation_6_Test_Shared/Documentation/Media/Badge.razor (hunk 1 above)
Comment (maxgsamson) [reply Blazor_Foundation_6_Test_Shared/Documentation/Media/Badge.razor:15]: Remove . before class

**Pull request closed without merging**